staging/src/k8s.io/client-go/Godeps/Godeps.json

```diff
@@ -1,16 +1,12 @@
 {
 	"ImportPath": "k8s.io/client-go",
-	"GoVersion": "go1.9",
+	"GoVersion": "go1.8",
```

Comment:
I doubt you should be touching those files.

Reply:
I think this got pulled in accidentally by a revase.